ray_on_golem/server/settings.py

```diff
@@ -31,7 +31,7 @@
 URL_TERMINATE_NODE = "/terminate"
 URL_GET_SSH_PROXY_COMMAND = "/ssh_proxy_command"
 URL_GET_OR_CREATE_DEFAULT_SSH_KEY = "/ger_or_create_default_ssh_key"
-URL_SELF_SHUTDOWN = "/self_shutdown"
+URL_SHUTDOWN = "/shutdown"
```

Comment:
If we are changing terms and removing the `self` part, let's do it globally.

Reply:
if you mean updating every piece of code to remove the word "self" from there, I think it's too much here -> this can be done with a separate pull request if indeed needed - what I wanted to have here is a cleaner API (where the word "self" doesn' really give any meaning) but I don't see how that implies fixing _all_ the code that contains it right now ...

Reply:
I mean, of course it can be done, I just think that putting such remarks on pull requests doesn't serve us well and hinders our progress more than helps it...

Reply:
anyway, @approxit I just updated the name of the view to match the URL -> other places actually mean what they should -> iow, they enable ability to _self_ shutdown